geoh5py/groups/property_group.py

```diff
@@ -60,6 +71,24 @@ def __init__(self, parent: ObjectBase, **kwargs):
             except AttributeError:
                 continue
 
+        self.parent.workspace.register_property_group(self)
+
+    def add_properties(self, data: Data | list[Data]):
```

Comment:
I am not sure about this but should the data already present on a workspace aren't they?
So accepting uid and name to add properties could be uselfull?

Reply:
Same as above - cool with uuid, probably not name.